run/idp-sql/test/retry.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
```

Comment:
nit: More universal for the rare environment where default bash isn't /bin/bash.

```suggestion
#!/usr/bin/env bash
```